index.html

```diff
@@ -11,6 +11,7 @@
 </head>
 
 <body>
+    <h1>hey</h1>
```

Comment:
Hello @98arun , Can we add a wrapper or div container above `h1` tag to wrap the element in a proper parent element

Reply:
Hello @xDAnkit, that would be a good approach. Let me do that